content/blog/2022-03-31-training-and-saving-models-on-a-dedicated-runner.md

```diff
@@ -0,0 +1,377 @@
+---
+title: Training and saving models with CML on a dedicated runner
+date: 2022-03-31
+description: Hello World
+descriptionLong: |
+  Some long
+  multiline
+  text (supports _basic_ [Markdown](link))
```

Comment:
Desc and summary are important 🙂 
You can see the former in the article thumb https://dvc-org-application-ass-fza3m8.herokuapp.com/blog and the latter on the top of the article plage https://dvc-org-application-ass-fza3m8.herokuapp.com/blog/training-and-saving-models-on-a-dedicated-runner .

Reply:
Also `pictureComment` can be removed if not used.

Reply:
Thanks, provided the metadata now!